samples/CommunityToolkit.Maui.Sample/ViewModels/Essentials/SpeechToTextViewModel.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Globalization;
+using CommunityToolkit.Maui.Alerts;
+using CommunityToolkit.Maui.Media;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Essentials;
+
+public partial class SpeechToTextViewModel : BaseViewModel
+{
+	const string defaultLanguage = "en-US";
+
+	readonly ITextToSpeech textToSpeech;
+	readonly ISpeechToText speechToText;
+
+	[ObservableProperty]
+	Locale? locale;
+
+	[ObservableProperty]
+	string recognitionText = "Welcome to .NET MAUI Community Toolkit!";
+
+	public SpeechToTextViewModel(ITextToSpeech textToSpeech, ISpeechToText speechToText)
+	{
+		this.textToSpeech = textToSpeech;
+		this.speechToText = speechToText;
+
+		Locales.CollectionChanged += HandleLocalesCollectionChanged;
+	}
+
+	public ObservableCollection<Locale> Locales { get; } = new();
+
+	[RelayCommand]
+	async Task SetLocales()
+	{
+		Locales.Clear();
+
+		var locales = await textToSpeech.GetLocalesAsync();
+		foreach (var locale in locales.OrderBy(x => x.Language).ThenBy(x => x.Name))
+		{
+			Locales.Add(locale);
+		}
+
+		Locale = Locales.FirstOrDefault(x => x.Language is defaultLanguage) ?? Locales.FirstOrDefault();
+	}
+
+	[RelayCommand]
+	async Task Play(CancellationToken cancellationToken)
+	{
+		await textToSpeech.SpeakAsync(RecognitionText, new()
```

Comment:
Did you see it crash? 

`RecognitionText` is a non-nullable string, so it _shouldn't_ ever be null (famous last words...)

Reply:
Yes, I see... When you press the `Listen` button and there's nothing there and you press the `Play` button it will crash, when I come back to my PC I can record a video